SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -1018,16 +1024,23 @@ void FSpatialGDKEditorToolbarModule::OpenInspectorURL()
 	}
 }
 
-void FSpatialGDKEditorToolbarModule::LaunchInspectorWebpageButtonClicked()
+void FSpatialGDKEditorToolbarModule::StartInspectorProcess(TFunction<void()> InOnReady)
 {
+	auto OnReady = [InOnReady]() {
```

Comment:
Why do we need to wrap the callback in another callback? We can test InOnReady directly?

Reply:
do you think that would be more readable/this is a bad pattern? It would be in two places! this is less verbose in js where I basically got it from though (though in js this would be even cleaner with es2020's optional  chaining `InOnReady.?()`)

Reply:
Hmm I see your point 1 line vs 5 but I personally think the indirection is too high a price to pay in this case, I don't really love calling a lambda that hides whether it actually executes the lambda it wraps. Its already a tricky bit of code I'd prefer to inline this part of the logic. Also however sad it is that cxx doesnt have `?` operator `if (valid) valid();` is a common enough pattern not to have to be abstracted away. Anyway we are pretty deep into code style territory, I leave it up to you.

Reply:
I think you make valid points, I've changed it to similar to what you suggested